documentation/source/user_section/tutorials/4d/processing-dmri-data/computing-dti.rst

```diff
@@ -0,0 +1,30 @@
+Computing DTI for motion corrected dMRI data
+############################################
+
+Here, we compute the diffusion tensor. SCT relies on the excellent ``dipy`` library for computing dMRI metrics `[Garyfallidis et al., Front Neuroinform 2014] <https://pubmed.ncbi.nlm.nih.gov/24600385/>`_.
+
+.. code::
+
+   sct_dmri_compute_dti -i dmri_crop_moco.nii.gz -bval bvals.txt -bvec bvecs.txt
+
+:Input arguments:
+   - ``-i`` : The input dMRI image.
+   - ``-bval`` : A text file containing a b-value for each volume in the dMRI image, indicating the diffusion weightings for each of the volumes in the dMRI image.
+   - ``-bvec`` : A text file with three lines, each containing a value for each volume in the input image. Together, the the three sets of values represent the ``(x, y, z)`` coordinates of the b-vectors, which indicate the direction of the diffusion encoding for each volume of the dMRI image.
+
+.. note::
+
+   You can also supply the ``-method restore`` option to estimate the tensors using **"RESTORE: robust estimation of tensors by outlier rejection"** `[Chang, Magn Reson Med 2005] <https://pubmed.ncbi.nlm.nih.gov/15844157/>`_.
+
+:Output files/folders:
+   - ``dti_FA.nii.gz`` : Fractional anisotropy (FA) diffusion tensor image.
+   - ``dti_AD.nii.gz`` : Axial diffusivity (AD) diffusion tensor image.
+   - ``dti_MD.nii.gz`` : Mean diffusivity (MD) diffusion tensor image.
+   - ``dti_RD.nii.gz`` : Radial diffusivity (RD) diffusion tensor image.
+
+.. TODO: Why doesn't sct_dmri_compute_dti output "Geodesic anisotropy (GA)"?
```

Comment:
I was digging into the `dipy` source code to try to understand FA/AD/MD/RD, and noticed that there's a metric we don't use: [geodesic ansiotropy](https://github.com/dipy/dipy/blob/c846bf5a23b7e95343a9cf231df2653473602456/dipy/reconst/dti.py#L92). Is GA something that would be useful to expose? (I don't know enough to tell if this is useful.)

Reply:
> Is GA something that would be useful to expose? 

no. <1% of researchers use GA, and the other 99% will get confused